x/oracle/abci.go

```diff
@@ -14,6 +14,13 @@ import (
 func EndBlocker(ctx sdk.Context, k keeper.Keeper) error {
 	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyEndBlocker)
 
+	// Check for Oracle parameter update plans and execute it if one is
+	// found and the update plan height is set to the current block.
+	plan, found := k.GetParamUpdatePlan(ctx)
```

Comment:
Looks like it is written to only expect one update plan at a time. Is there any use case where we would want multiple queued up at once?

Reply:
The only one I can think of is if we want to schedule multiple asset listings at different dates. Otherwise if we wanted to change the updates of an existing plan having multiple wouldn't make sense. @adamewozniak wdyt, do you see scenarios where we have price feeds tested and confirmed for multiple assets already and want to list them at separate dates rather than all together when they're all ready?

Reply:
I think it could be possible yeah, but not a necessary feature - this is mostly to keep us & validators from waking up in the middle of the night to put a new price feed config up